.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
This also seems worth looking at, because this PR has over 600,000 additions, and 8421 files...

Reply:
I noticed this also while was creating the PR but this could be a very good question for Bob, maybe?